code/datums/wounds/types/slashes.dm

```diff
@@ -61,7 +61,7 @@
 	/// Organs we can disembowel associated with chance to disembowel
 	var/static/list/affected_organs = list(
 		ORGAN_SLOT_STOMACH = 100,
-		ORGAN_SLOT_LIVER = 50,
+		ORGAN_SLOT_LIVER = 100,
```

Comment:
Why this change?

Reply:
Consistency, now it's guaranteed to be 2 organs rather than a chance at 2